src/relativePxth.ts

```diff
@@ -0,0 +1,52 @@
+import invariant from 'tiny-invariant';
+
+import { createPxth } from './createPxth';
+import { getPxthSegments } from './getPxthSegments';
+import { isInnerPxth } from './isInnerPxth';
+import { Pxth } from './Pxth';
+import { pxthToString } from './pxthToString';
+import { RootPathToken } from './RootPath';
+import { samePxth } from './samePxth';
+
+const getStringifiedPxth = (pxth: Pxth<unknown>): string => {
+    const path = pxthToString(pxth);
+
+    if (path === RootPathToken) {
+        return 'RootPathToken';
+    }
+
+    return path;
+};

```

Comment:
Why this function is needed? You can attach a string for the symbol, for more descriptive name
https://github.com/ArtiomTr/pxth/blob/a504b8226144c09b54adeda12eeb7906597ca45e/src/RootPath.ts#L1
And then, you could simply call `pxthToString(pxth).toString()`, to ensure, that it won't throw error.

Because at this moment, this function is confusing - what is the difference between `pxthToString` and `getStringifiedPxth`? Their names have same meaning.